CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.7.5
+
+* Fix regex for footnotes in call-to-action boxes [222](https://github.com/alphagov/govspeak/pull/222)
```

Comment:
Suggested tweak as we haven't just fixed a regex - we've also defined a `extension("call-to-action")`.

```suggestion
* Fix footnotes in call-to-action boxes [222](https://github.com/alphagov/govspeak/pull/222)
```